### DISCUSSION TITLE
Project Shutdown - looking for another maintainer

### DISCUSSION THREAD

**Discussion by bcssov** [bcssov/IronyModManager#425] [General]
Irony was developed at first as a tool to help me manage my Stellaris gameplay which I decided to share with the world. It grew into the toolkit that you see today. 

After 3 years of maintaining Irony I've decided to step down from the project. I've stopped playing any PDX games over a year ago and have maintained this project simply for fun at first, but in recent months this has become a chore. I simply cannot dedicate anymore spare time to this project related to game(s) that I don't even play. Honestly speaking I lost my motivation for this project.

Hence the sad news that this project will be shutdown unless a maintainer is found. At this time nobody came forth; so the future of the project is in the balance. 

Irony will continue to work but over time some features might break down:
1. Stellaris\HOI4 conflict solving as new game updates are released. This does not mean that the feature will break down it means that Irony will no longer be able to reliably tell which definition will be used by the game or properly resolve conflict overrides.
2. Over time if paradox ever makes any changes to its own Paradox Launcher sqlite db schema Irony will cease to function alongside PDX Launcher making it unable to export the playsets there when you hit apply\launch game.

It's been a fun ride, if you're interested to take over maintenance reply here or hop onto discord and let me know over there.

**Comment by Idhrendur** [comment]
Thanks for all your work on this over the last few years!

**Comment by SonGorLick** [comment]
If you havent already make a post in the stellairs modding den or if your in the crusader kings 3 co-op server post in there as well saying your looking for someone to maintaine this.

**Comment by Az0na** [comment]
Shame this tool made so many issues go away. notice recently applying errors but no point reporting them as support and maintance is gonna end anyway. thank you for making this and hopefully some one will take over the tourch if not this will be missed.

**Reply by SonGorLick** [reply]
Yeah im hoping so to,i may look into seeing if i know anyone who could do it or seeing if i can do it myself.

**Comment by bcssov** [comment]
PDX changing db schema (or in other words breaking compatibility) so soon is unexpected.
